cat-test/lib/cat_cafe.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:math' show Random;
+
+// Let us feed your kitty!
+class CatCafe {
+  static final _index = new Random();
```

Comment:
Not sure about the naming here. I usually name Random instances something like `_random`, although that's also probably not ideal. No big deal either way.